### PR TITLE
[Model Update] Get production forecast 2.0.0

### LINKED ISSUE #533: [Model Update]: Get Production Forecast
## Update Reason

The Get Production Forecast data model is being expanded due to the use of the updated header model

https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.shopfloor_information.get_production_forecast/1.0.0/GetProductionForecast.ttl

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [ ] The potential update will be either 
  - [ ] backward compatible or 
  - [x] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by FlorianDue**
## Description

Message Header and shopfloor_information_types are updated to version 2.0.0updated

 -->

Closes #533 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.shopfloor_information.get_production_forecast/2.0.0/GetProductionForecast.ttl [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
```
Comment (agg3fe) [inline io.catenax.shopfloor_information.get_production_forecast/2.0.0/GetProductionForecast.ttl:16]: please change to samm:meta-model:2.1.0

**Comment by FlorianDue** [comment]
Thank you for your feedback!

I just pushed the latest version and similar to the shopfloor_information_types, I set the samm metamodel to 2.1.0, removed the spaces at the end and set version 1.0.0 to deprecated.

Looking forward to your feedback!

File: io.catenax.shopfloor_information.get_production_forecast/2.0.0/GetProductionForecast.ttl [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    samm:description "Entity for a request of a production forecast"@en;
+    samm:properties (:offset :customerId [ samm:property :precisionOfForecast; samm:optional "true"^^xsd:boolean ] :productionForecastForAll :orderId [ samm:property :deviationOfSchedule; samm:optional "true"^^xsd:boolean ] [ samm:property :notificationInterval; samm:optional "true"^^xsd:boolean ] ext-types2:communicationMode ext-header2:version).
+
+:offset a samm:Property;
+    samm:preferredName "offset"@en;
+    samm:description "Send/start time of the first message/notification\n- \"0\" ==> immediate response"@en;
+    samm:characteristic ext-types2:TimeValueCharacteristic.
+
+:customerId a samm:Property;
+    samm:preferredName "customerId"@en;
```
Comment (agg3fe) [inline io.catenax.shopfloor_information.get_production_forecast/2.0.0/GetProductionForecast.ttl:57]: preferredName should be human readable and follow normal orthography (e.g., no camel case but normal word separation)

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

**Comment by agg3fe** [comment]
@FlorianDue 
since model conains reference to shopfloorinformationtypes, I would merge this model after the referenced model.

**Comment by FlorianDue** [comment]
> please check my comments
should be fixed

**Comment by FlorianDue** [comment]
> @FlorianDue since model conains reference to shopfloorinformationtypes, I would merge this model after the referenced model.
Thank you for your support!

That's fine. We would also like to get MS3 on Monday for these 3 models (#553  #554 and #551) Are there any steps that we have to do? For example create a new issue?

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by agg3fe** [comment]
> > @FlorianDue since model conains reference to shopfloorinformationtypes, I would merge this model after the referenced model.
> > Thank you for your support!
> 
> That's fine. We would also like to get MS3 on Monday for these 3 models (#553 #554 and #551) Are there any steps that we have to do? For example create a new issue?

Nothing else is required. We are good for MS3 in next meeting. please make sure someone from your team is there in meeting on Monday to present the model.

**Comment by FlorianDue** [comment]
Sure, ill be there. See you Monday and have a nice weekend!

**Pull request merged**